config/testdata/invalidConfig.toml

```diff
@@ -0,0 +1,2 @@
+
+Hello world!
```

Comment:
this file needs to be removed if we decide to remove the test for `InvalidTomlConfigFile`